src/System.Text.Http.Parser/BadHttpRequestException.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace System.Text.Http.Parser
+{
+    internal static class StatusCodes
+    {
```

Comment:
We seem to be missing some of these in https://github.com/dotnet/corefx/blob/master/src/System.Net.Primitives/src/System/Net/HttpStatusCode.cs#L8
Ideally we'd just add the missing values (https://github.com/dotnet/corefx/issues/4382) and then use that.